src/SDKs/_metadata/apimanagement_resource-manager.txt

```diff
@@ -1,11 +1,69 @@
-2018-05-11 16:48:52 UTC
+﻿Installing AutoRest version: latest
 
+> autorest@2.0.4280 preinstall C:\Users\Sasolank\AppData\Roaming\npm\node_modules\autorest
+> node ./preinstall-check
+
+C:\Users\Sasolank\AppData\Roaming\npm\autorest -> C:\Users\Sasolank\AppData\Roaming\npm\node_modules\autorest\dist\app.js
++ autorest@2.0.4280
+updated 1 package in 1.583s
+
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/apimanagement/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\github\azure-sdk-for-net\src\SDKs
+AutoRest code generation utility [version: 2.0.4280; node: v8.9.3]
+(C) 2018 Microsoft Corporation.
+https://aka.ms/autorest
+   Loading AutoRest core      'C:\Users\Sasolank\.autorest\@microsoft.azure_autorest-core@2.0.4280\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4280)
+   Loading AutoRest extension '@microsoft.azure/autorest.csharp' (~2.2.51->2.2.57)
+   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.43->2.3.43)
+WARNING: "HEAD 'GetEntityTag' method missing 404 status code section -- this might be unintentional.
```

Comment:
@fearthecowboy any reason why AutoRest is emitting these warnings?
@shahabhijeet FYI